Egress filtering now support FQDN and custom default Policy (#1646)
platformsh/platformsh-docs: docs/src/configuration/app/firewall.md

If no `ports` property is specified, requests to any port on the specified IP addresses are permitted.

### `domains`

This property is an array of fully qualified domain names [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name).  FQDN allows you to specify specific destinations by hostnames.

> If no `ports` property is specified, requests to any port on the specified IP addresses obtained during DNS names resolution for the specified FQDNs are permitted.

Reviewer: ```suggestion
If no `ports` property is specified, requests to any port on IP addresses the listed domains resolve to are permitted.
```

Requests to port 80 on any IP will be allowed, and requests to 1.2.3.4 on either port 80 or 443 will be allowed, even though the first rule only lists port 443.

## Default Outbound Restrictions Policy

By default outbound rules have an `allow` policy, which means that if no outbound rules is set, all TCP traffic (except for port 25) is allowed.

> The default policy can be set to `deny`. File a support ticket and ask the support team to do so.

Reviewer: ```suggestion
The default policy can be set to `deny` by request. Please file a support ticket and ask the support team to do so.
```

Requests to port 80 on any IP will be allowed, and requests to 1.2.3.4 on either port 80 or 443 will be allowed, even though the first rule only lists port 443.

## Default Outbound Restrictions Policy

By default outbound rules have an `allow` policy, which means that if no outbound rules is set, all TCP traffic (except for port 25) is allowed.

The default policy can be changed on a project basis, patching the `outbound_restrictions_default_policy` setting accordingly: `allow | deny`.

```bash
platform p:curl -X PATCH --project PROJECT_ID /settings -d '{"outbound_restrictions_default_policy": "deny"}'
```

> NOTE: This is an operation permitted only to platform admins.

Reviewer: If only Platform employees can do so, it should be documented in playbook, not here.  If anything, the public docs should just say "can be set to deny.  File a support ticket and ask the support team to do so."
